target/scripts/check-for-changes.sh

```diff
@@ -5,48 +5,40 @@
 
 # shellcheck source=./helpers/index.sh
 source /usr/local/bin/helpers/index.sh
+# determine `POSTMASTER_ADDRESS` for alias (re-)generation
+# shellcheck source=/dev/null
+source /etc/dms-settings
 
-function _log_date
-{
-  date +"%Y-%m-%d %H:%M:%S"
-}
+_log 'debug' "$(_log_date) Starting changedetector"
 
-LOG_DATE=$(_log_date)
-_notify 'task' "${LOG_DATE} Start check-for-changes script."
-
-# ? --------------------------------------------- Checks
-
-cd /tmp/docker-mailserver || exit 1
+if ! cd /tmp/docker-mailserver &>/dev/null
+then
+  _log "Could not change into '/tmp/docker-mailserver/' directory"
+  exit 1
+fi
 
 # check postfix-accounts.cf exist else break
 if [[ ! -f postfix-accounts.cf ]]
 then
-  _notify 'inf' "${LOG_DATE} postfix-accounts.cf is missing! This should not run! Exit!"
-  exit 0
+  _log 'error' "'/tmp/docker-mailserver/postfix-accounts.cf' is missing"
+  exit 1
 fi
 
 # verify checksum file exists; must be prepared by start-mailserver.sh
 if [[ ! -f ${CHKSUM_FILE} ]]
 then
-  _notify 'err' "${LOG_DATE} ${CHKSUM_FILE} is missing! Start script failed? Exit!"
-  exit 0
+  _log 'error' "'/tmp/docker-mailserver/${CHKSUM_FILE}' is missing"
+  exit 1
 fi
```

Comment:
`_exit_with_error` is probably more suitable here? It'd retain the same behaviour but also include the "Aborting" message that you've dropped?

Reply:
> `_exit_with_error` is probably more suitable here? It'd retain the same behaviour but also include the "Aborting" message that you've dropped?

👍🏼

Reply:
Done in 5f71f074fca7f5b1eb44978206c9f589065f7799